Update client-firewall-rules.md (#150)
microsoft/containerregistry: docs/client-firewall-rules.md

![Azure Application Rule](/media/mcr-client-firewall-rules.png)

## Service Tag Usage

> Microsoft Artifact Registry uses [Azure Front Door](https://learn.microsoft.com/en-us/azure/frontdoor/front-door-overview) as its entry point. As such if you are using service tag for firewall configuration you should enable both MicrosoftContainerRegistry as well as the AzureFrontDoor.FirstParty [service tags](https://learn.microsoft.com/en-us/azure/virtual-network/service-tags-overview)

Reviewer: ```suggestion
Microsoft Artifact Registry uses [Azure Front Door](https://learn.microsoft.com/en-us/azure/frontdoor/front-door-overview) as its entry point. If using service tags for firewall configuration, **both MicrosoftContainerRegistry and AzureFrontDoor.FirstParty [service tags](https://learn.microsoft.com/en-us/azure/virtual-network/service-tags-overview#available-service-tags) should be enabled.**
```
